app/styles.module.css

```diff
@@ -13,4 +13,73 @@
   max-width: 350px;
   height: 100%;
   border: 1px solid black;
+  padding: 10px;
+}
+.headingInfo {
+  display: flex;
+  flex-direction: row;
+}
+.descriptionText {
+  font-size: 12px;
+  font-weight: 200;
+}
+div.headingInfo p {
+  font-size: 11px;
+  font-weight: 600;
+  padding-top: 10px;
+  padding: 5px;
+}
+
+div.headingInfo p:last-of-type {
+  margin-left: auto;
+  color: lightgray;
+  font-size: 12px;
+  font-weight: normal;
+}
+
+.post {
+  display: flex;
+  flex-direction: column;
+  font-weight: 200;
+}
+
+.postInteractions {
+  margin-top: 3%;
+  display: flex;
+  flex-direction: row;
+  padding: 5px;
+}
+.postInteractions > p {
+  font-size: 11px;
+  font-weight: 600;
+  padding: 1%;
+  margin-left: 1%;
+}
+.postInteractions svg:last-of-type {
+  margin-left: auto;
+}
+.content > p {
+  font-weight: 200;
+}
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  height: 60px;
+  background: #fafafa;
+  margin: -10px;
+  margin-bottom: 10px;
+}
+.header > h1:first-of-type {
+  color: #0078e8;
+}
+.header > h1 {
+  font-size: 14px;
+  font-weight: 600;
+}
+
+.comment {
+  padding-left: 30px;
 }
```

Comment:
Consider whether it's possible to have the username and comment text in the same div, that way they're always in line with each other.